src/models/StuntSheet.ts

```diff
@@ -96,4 +99,69 @@ export default class StuntSheet extends Serializable<StuntSheet> {
       }
     }
   }
+
+  /**
+   * Calculates warnings associated with this stunt sheet
+   */
+  calculateWarnings(ss: number): void {
+    this.warnings = [];
+
+    // Ensure no dots are too close
+    this.stuntSheetDots.forEach((dot1) => {
+      this.stuntSheetDots.forEach((dot2) => {
+        // Spooky O(n^2)...
```

Comment:
Thanks for the callout here. This gets recalculated each time a dot gets moved, correct? Have you checked if this causes any noticeable performance hits on full-scale shows?

Reply:
I couldn't find any noticeable dips in performance when I tried with Miley show (178 dots).